### PR TITLE
[F] Ported OwnerContentResource to spec-first (ENT-1878)

### PR DESCRIPTION
**Pull request by AmitMandliya**
- Changed the method name listContent to listOwnerContent and getContent to getOwnerContent
- Changed the OwnerContentResourceTest file to test the right methods as above
- Made changes in candlepin-api-spec for OwnerContentResource

### CODE REVIEW

File: server/src/main/java/org/candlepin/resource/OwnerContentResource.java [hunk 1]
```diff
@@ -226,12 +199,8 @@ private Content createContentImpl(Owner owner, ContentDTO content) {
...
     }
 
-    @ApiOperation(notes = "Creates a Content", value = "createContent")
-    @POST
-    @Produces(MediaType.APPLICATION_JSON)
-    @Consumes(MediaType.APPLICATION_JSON)
-    public ContentDTO createContent(@PathParam("owner_key") String ownerKey,
-        @ApiParam(name = "content", required = true) ContentDTO content) {
+    @Override
+    public ContentDTO createContent(String ownerKey, ContentDTO content) {
```
[the remark this answers is not in the document]
Comment (AmitMandliya) [reply server/src/main/java/org/candlepin/resource/OwnerContentResource.java:190]: I have added validations in ContentDTO, the unit tests ran fine in my machine. However, the PR is failing with unittests:

`[1BCreating network "unit-tests-jenkins-candlepin-candlepin-pullrequest-unittests-3264_default" with driver "bridge"
Couldn't connect to Docker daemon at http+docker://localhost - is it running?`

I am looking into the issue.

File: server/src/main/java/org/candlepin/resource/OwnerContentResource.java [hunk 2]
```diff
@@ -51,26 +44,13 @@
...
  * Manage the content that exists in an organization.
  */
-@Path("/owners/{owner_key}/content")
-@Api(value = "owners", authorizations = { @Authorization("basic") })
-public class OwnerContentResource {
+
+public class OwnerContentResource implements OwnersApi {
     private static Logger log = LoggerFactory.getLogger(OwnerContentResource.class);
 
     private ContentCurator contentCurator;
```
[the remark this answers is not in the document]
Comment (AmitMandliya) [reply server/src/main/java/org/candlepin/resource/OwnerContentResource.java]: I have removed the unused dependencies..

File: server/src/main/java/org/candlepin/resource/OwnerContentResource.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (AmitMandliya) [reply server/src/main/java/org/candlepin/resource/OwnerContentResource.java:190]: Validations has been added in update and createBatchContent methods.

**Comment by Ceiu** [comment]
As a note, we should avoid merging these PRs before they're squashed, unless we only want to see force-pushes on every change request.

**Pull request merged**